src/backend/settings/__init__.py

```diff
@@ -153,6 +147,11 @@
 ]
 
 TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.jinja2.Jinja2",  # TODO: This is terrible
+        "DIRS": [os.path.join(BASE_DIR, "templates")],
+        "APP_DIRS": True,
+    },
```

Comment:
Why are you changing the template backend?

Reply:
This is because our existing templates are Jinja2 and carried over from mail-usv. Porting these to Django templating would be good but I considered it beyond the scope of this PR. We already do this within core.

https://github.com/ractf/core/blob/000fa7ed52857841d8532ce0e1b34d5f87232037/src/backend/settings/production.py#L36-L43